doc/2/api/controllers/server/capabilities/index.md

```diff
@@ -0,0 +1,73 @@
+---
+code: true
+type: page
+title: capabilities
+---
+
+# Capabilities
```

Comment:
Missing the `SinceBadge`. You can use the `auto-version` placeholder, it will be replaced in the release process